spec/react_on_rails/utils_spec.rb

```
@@ -6,14 +6,126 @@
+      allow(ReactOnRails::PackerUtils.packer).to receive_message_chain("dev_server.protocol")
+        .and_return("http")
+      allow(ReactOnRails::PackerUtils.packer).to receive_message_chain("dev_server.host_with_port")
+        .and_return("localhost:3035")
+    end
+
     context "when server_bundle_path cleared" do
       before do
         allow(Rails).to receive(:root).and_return(File.expand_path("."))
         described_class.instance_variable_set(:@server_bundle_path, nil)
+        described_class.instance_variable_set(:@rsc_bundle_path, nil)
       end
```

rubocop (1.69.1)

[convention] Do not define multiple `before` hooks in the same example group (also defined on line 126).

(RSpec/ScatteredSetup)